platforms/web/src/containers/AppRoutes/AppRoutes.tsx

```diff
@@ -13,30 +13,89 @@ import PlaylistScreenRouter from '@jwp/ott-ui-react/src/pages/ScreenRouting/Play
+const EditPassword = React.lazy(() => import('@jwp/ott-ui-react/src/containers/AccountModal/forms/EditPassword'));
+const UpgradeSubscription = React.lazy(() => import('@jwp/ott-ui-react/src/components/UpgradeSubscription/UpgradeSubscription'));
+const PaymentFailed = React.lazy(() => import('@jwp/ott-ui-react/src/components/PaymentFailed/PaymentFailed'));
+const DeleteAccountModal = React.lazy(() => import('@jwp/ott-ui-react/src/components/DeleteAccountModal/DeleteAccountModal'));
+const DeleteAccountPasswordWarning = React.lazy(() => import('@jwp/ott-ui-react/src/components/DeleteAccountPasswordWarning/DeleteAccountPasswordWarning'));
+const UpdatePaymentMethod = React.lazy(() => import('@jwp/ott-ui-react/src/containers/UpdatePaymentMethod/UpdatePaymentMethod'));
+
 export default function AppRoutes() {
   const { t } = useTranslation('error');
 
   return (
-    <Routes>
-      <Route element={<RoutesContainer />}>
-        <Route element={<Layout />} errorElement={<RootErrorPage />}>
-          <Route index element={<Home />} />
-          <Route path={PATH_PLAYLIST} element={<PlaylistScreenRouter type={APP_CONFIG_ITEM_TYPE.playlist} />} />
-          <Route path={PATH_CONTENT_LIST} element={<PlaylistScreenRouter type={APP_CONFIG_ITEM_TYPE.content_list} />} />
-          <Route path={PATH_MEDIA} element={<MediaScreenRouter />} />
-          <Route path={PATH_LEGACY_SERIES} element={<LegacySeries />} />
-          <Route path={PATH_SEARCH} element={<Search />} />
-          <Route path={PATH_USER} element={<User />} />
-          <Route path={PATH_ABOUT} element={<About />} />
-          <Route
-            path="/*"
-            element={<ErrorPage title={t('notfound_error_heading', 'Not found')} message={t('notfound_error_description', "This page doesn't exist.")} />}
-          />
+    <>
+      <Routes>
+        <Route element={<RoutesContainer />}>
+          <Route element={<Layout />} errorElement={<RootErrorPage />}>
+            <Route index element={<Home />} />
+            <Route path={PATH_PLAYLIST} element={<PlaylistScreenRouter type={APP_CONFIG_ITEM_TYPE.playlist} />} />
+            <Route path={PATH_CONTENT_LIST} element={<PlaylistScreenRouter type={APP_CONFIG_ITEM_TYPE.content_list} />} />
+            <Route path={PATH_MEDIA} element={<MediaScreenRouter />} />
+            <Route path={PATH_LEGACY_SERIES} element={<LegacySeries />} />
+            <Route path={PATH_SEARCH} element={<Search />} />
+            <Route path={PATH_USER} element={<User />} />
+            <Route path={PATH_ABOUT} element={<About />} />
+            <Route
+              path="/*"
+              element={<ErrorPage title={t('notfound_error_heading', 'Not found')} message={t('notfound_error_description', "This page doesn't exist.")} />}
+            />
+          </Route>
         </Route>
-      </Route>
-    </Routes>
+      </Routes>
+      <AccountModal>
```

Comment:
Render function only for close handlers?
```suggestion
      <AccountModal>
        ({ handleClose ) => (
           <AccountModalRoute param="payment-error" element={<PaymentFailed type="error" message="todo" onCloseButtonClick={handleClose} />} />
        )
      </AccountModal>
```

Reply:
My goal is to remove most dynamic props from this file.